### PR TITLE
feat(api): add email configuration

### PR DESCRIPTION
**Pull request by jooola**
Related to #724 it will be fixed once we move the authentication/password reset to Django

### CODE REVIEW

**Comment by jooola** [comment]
Woups, this is missing a lot of settings, e.g. the From email address, and maybe other stuff.

File: api/libretime_api/settings/prod.py [hunk 1]
```diff
@@ -73,3 +73,20 @@
...
+# https://docs.djangoproject.com/en/4.2/topics/email/
+
+EMAIL_BACKEND = "django.core.mail.backends.smtp.EmailBackend"
+
+EMAIL_HOST = CONFIG.email.host
+EMAIL_PORT = CONFIG.email.port
+EMAIL_HOST_USER = CONFIG.email.user
+EMAIL_HOST_PASSWORD = CONFIG.email.password
+EMAIL_USE_TLS = CONFIG.email.security == "explicit"
+EMAIL_USE_SSL = CONFIG.email.security == "implicit"
```
Comment (jooola) [inline api/libretime_api/settings/prod.py:87]: `use_tls` or `use_ssl` are somewhat difficult to understand, so I changed this to use one of `None|implicit|explicit`.

- None is no tls encryption
- implicit is tls encryption
- explicit is tls encryption using starttls

Comment (paddatrapper) [reply api/libretime_api/settings/prod.py:87]: Hmm. I would maybe change the options to `None|SSL/TLS|STARTTLS` - those seem to be more consistent other email setting implementations (for example email clients, Nextcloud)

Comment (jooola) [reply api/libretime_api/settings/prod.py]: Yes, the options you proposed are better! I changed it.

**Pull request merged**